osu.Game.Resources/Shaders/sh_TriangleBorder.fs

```diff
@@ -13,33 +13,27 @@ layout(std140, set = 0, binding = 0) uniform m_BorderData
 
 layout(location = 0) out vec4 o_Colour;
 
-highp float dstToLine(highp vec2 start, highp vec2 end, highp vec2 pixelPos)
+// Distance to the line with the points (0.0, 1.0) and (0.5, 0.0)
+highp float dstToSideLine(highp vec2 pixelPos)
 {
-    highp float lineLength = distance(end, start);
-
-    if (lineLength < 0.001)
-        return distance(pixelPos, start);
-
-    highp vec2 a = (end - start) / lineLength;
-    highp vec2 closest = clamp(dot(a, pixelPos - start), 0.0, lineLength) * a + start; // closest point on a line from given position
+    highp vec2 a = vec2(0.4472, -0.8944);
```

Comment:
How is this calculated?

Reply:
Formula is the same `(end - start) / distance(end, start)`, where start and end points are in the comment above.

Reply:
Certainly. There are no pointers as to how the value was calculated from a first glimpse at the code.